requirements/product-environment/product-environment.tex

```diff
@@ -0,0 +1,41 @@
+\section{Product Environment}
+  \subsection{User Client}
+    \subsubsection{Hardware Requirements}  
+      \begin{itemize}
+      \item A connection to a LAN with a central server
+      \item 2 MB of storage space and additional 100MB per image
+      \end{itemize}
+    \subsubsection{Software Requirements}  
+      \begin{itemize}
+      \item None, so far
+    \end{itemize}
+
+  \subsection{Central Server}
+    \subsubsection{Hardware Requirements}  
+    \begin{itemize}
+      \item A connection to a LAN
+      \item 500 MB of storage space
+      \item 8 GB of RAM
+      \item One physical CPU core
+    \end{itemize}
+    \subsubsection{Software Requirements}  
+    \begin{itemize}
+      \item Linux Ubuntu 18.04 or newer
+      \item MySQL Database Server version 8.0 or newer
+      \item Linux kernel version 3.10 or higher
+      \item CS Docker engine 1.13, EE Deamon 17.03 and higher
```

Comment:
Python 3

Reply:
Should I also add all the external libraries we'll use?

Reply:
We'll be using `pip` so the user has to install it. I guess python3 is a dependency of `pip` anyway.